sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Trigger/WebPubSubTriggerDispatcher.cs

```diff
@@ -83,15 +81,37 @@ public async Task<HttpResponseMessage> ExecuteAsync(HttpRequestMessage req,
                     case RequestType.Connect:
                         {
                             var content = await req.Content.ReadAsStringAsync().ConfigureAwait(false);
-                            var request = JsonSerializer.Deserialize<ConnectEventRequest>(content);
-                            eventRequest = new ConnectEventRequest(context, request.Claims, request.Query, request.Subprotocols, request.ClientCertificates, request.Headers);
+                            if (context is MqttConnectionContext mqttContext)
+                            {
+                                var request = JsonSerializer.Deserialize<MqttConnectEventRequestDeserializationHelper>(content);
```

Comment:
naming: `MqttConnectEventRequestDeserializationHelper` helper is usually a static helper class

Reply:
`MqttConnectEventRequestContent`?